tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/service/impl/RevalidationRabbitServiceImpl.java

```diff
@@ -13,7 +13,7 @@ public class RevalidationRabbitServiceImpl implements RevalidationRabbitService
   @Value("${app.rabbit.reval.exchange}")
   private String exchange;
 
-  @Value("${app.rabbit.reval.routingKey.connection.update}")
+  @Value("${app.rabbit.reval.routingKey.revalupdate.created}")
```

Comment:
I don't see this routing key in TIS-RabbitMQ repo. @CaiWillis Am I missing something?